core/api/cart/features/closeMicrocart.js

```diff
@@ -2,14 +2,14 @@
  * [Helper] Optional UI functionality for closing microcart.
  *
  * #### Methods
- * - **`closeMicrocart`** sets `microcart` and `overlay` property from ui state to `false`. Dispatches `ui/setMicrocart'` Vuex action
+ * - **`closeMicrocart`** sets `microcart` and `overlay` property from features state to `false`. Dispatches `features/setMicrocart'` Vuex action
```

Comment:
`ui` instead of features was correct, this one applies to vuex store, not folder